iroha/src/sumeragi.rs

```diff
@@ -48,9 +55,10 @@ pub struct Sumeragi {
     sorted_peers: Vec<PeerId>,
     max_faults: usize,
     peer_id: PeerId,
-    /// Block in discussion this round
-    pending_block: Option<Block>,
+    /// PendingBlock in discussion this round
```

Comment:
Shouldn't it be `SignedBlock in discussion ...` here?

Reply:
https://soramitsu.atlassian.net/browse/RH2-86